README.md

```diff
@@ -248,12 +248,12 @@ Include a citation for Label Studio in the **References** section of your articl
     Mikhail Malyuk and
     Andrey Holmanyuk and
     Nikolai Liubimov},
-  year={2020-2024},
+  year={2020-2025},
 }
 ```
 
 ## License
 
-This software is licensed under the [Apache 2.0 LICENSE](/LICENSE) © [Heartex](https://www.heartex.com/). 2020-2024
```

Comment:
@deppp I assume this still points to Heartex for good reasons? -- or can we update it?